pkg/common/utils.go

```diff
@@ -174,14 +174,15 @@ func GetSharedServicesNamespace(ctx context.Context, name string) (namespace str
 }
 
 func MergeMap(in map[string]string, mergeMap map[string]string) map[string]string {
+	//in map is always Not NIL -- from the function calls, we ensure in map always has some data in it
```

Comment:
I would rather remove this line - this is already defensively coded such that the `nil` case is handled (either var is replaced with empty `map[string]string` in that event), but this comment conflicts with the logic.